02Arrays/Array/Tester.swift

```diff
@@ -73,7 +110,19 @@ class Tester {
     
     
```

Comment:
largestNum does not get reset if it it is smaller than previous largestNum
[7,3,10,1,3] should return 7